src/cloud-api-adaptor/podvm/qcow2/build-s390x-se-image-post.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [ "${SE_BOOT:-0}" != "1" ]; then
+if [ "${SE_BOOT:-false}" != "true" ]; then
```

Comment:
Hi @bpradipt, this was the change I made. it was like this earlier- if [ "${SE_BOOT:-0}" != "true" ]; then
 i made it this if [ "${SE_BOOT:-false}" != "true" ]; then